Lib/test/test_regrtest.py

```diff
@@ -0,0 +1,427 @@
+"""
+Tests of regrtest.py.
+
+Note: test_regrtest cannot be run twice in parallel.
+"""
+from __future__ import print_function
+
+import collections
+import os.path
+import platform
+import re
+import subprocess
+import sys
+import sysconfig
+import tempfile
+import textwrap
+import unittest
+from test import support
+
+
+Py_DEBUG = hasattr(sys, 'getobjects')
+ROOT_DIR = os.path.join(os.path.dirname(__file__), '..', '..')
+ROOT_DIR = os.path.abspath(os.path.normpath(ROOT_DIR))
+
+TEST_INTERRUPTED = textwrap.dedent("""
+    from signal import SIGINT
+    try:
+        from _testcapi import raise_signal
+        raise_signal(SIGINT)
+    except ImportError:
+        import os
+        os.kill(os.getpid(), SIGINT)
+    """)
+
+
+SubprocssRun = collections.namedtuple('SubprocssRun', 'returncode stdout, stderr')
```

Comment:
`'returncode stdout, stderr'` looks strange.

This namedtuple is used only in two places, and only its stdout attribute is used. Wouldn't be simpler to return just stdout from `run_command()`? Or at least return a 3-tuple?